virtual-env.ipynb

```diff
@@ -26,78 +31,116 @@
+    "Follow these steps from your Jupyter service to create a Python virtual environment using Conda: \n",
     "\n",
     "1.  <a id=\"conda-virt-env-setup-step-create-terminal\"></a>Create a new terminal by selecting the **New Launcher** option (`+` icon) from the top action toolbar in the left sidebar, and then selecting **Terminal** from the main work area.<br>\n",
-    "   The next steps should be executed from your new terminal, except where otherwise specified.\n",
-    "\n",
+    "   The next steps should be executed from your new terminal, except where otherwise specified.<br>\n",
     "\n",
     "2.  <a id=\"conda-virt-env-setup-step-create-env\"></a>Create a new Python virtual environment by running the following command.\n",
     "    Replace `<environment name>` with your preferred virtual-environment name:\n",
     "    ```sh\n",
-    "    conda create -n <environment name> ipykernel python=3.7\n",
+    "    conda create -n <environment name>\n",
     "    ```\n",
-    "\n",
-    "    <br>For example, the following command creates an environment named \"myenv\":<br>\n",
+    "    <br>For example, the following command creates an environment named \"myenv\":\n",
     "    ```sh\n",
-    "    conda create -n myenv ipykernel python=3.7\n",
+    "    conda create -n myenv\n",
     "    ```\n",
+    "    <br>\n",
     "\n",
-    "\n",
-    "3. <a id=\"conda-virt-env-setup-step-install-packages\"></a>Install the desired Python packages by running the following command.\n",
-    "   Replace `<environment name>` with the name of the environment that you created in the previous step and `<package>` with the name of your selected package, optionally followed by `=<version>`; you can specify multiple packages:\n",
+    "3. <a id=\"conda-virt-env-setup-step-install-packages\"></a>Activate the Conda environment by running the following command:\n",
     "   ```sh\n",
-    "   conda install -n <environment name> <package> [<package> ...]\n",
+    "   conda activate <environment name>\n",
     "   ```\n",
-    "\n",
-    "   <br>For example, the following command installs the SciPy, pandas version 0.25.0, and TensorFlow version 1.13.1 packages for a \"myenv\" environment:\n",
+    "   \n",
+    "   <br>For example, the following command activates an environment named \"myenv\":\n",
     "   ```sh\n",
-    "   conda install -n myenv scipy pandas=0.25.0 tensorflow=1.13.1 python=3.7\n",
-    "   ```\n",
+    "   conda activate myenv\n",
+    "   ```   \n",
+    "   <br>\n",
     "\n",
+    "4. <a id=\"conda-virt-env-setup-step-install-packages\"></a>Once you activate your environment, you can use either PIP or Conda to install the necessary Python packages.<br> To do this, replace `<package>` with the name of the package you want to install, and optionally add `==<version>` for PIP or `=<version>` for Conda. You can specify multiple packages in the same command.<br>\n",
```

Comment:
"4. <a id=\"conda-virt-env-setup-step-install-packages\"></a>Use either PIP or Conda to install the necessary Python packages.<br> To do this, replace `<package>` with the name of the package you want to install, and optionally add `==<version>` for PIP or `=<version>` for Conda. You can specify multiple packages in the same command.<br>\n",

Reply:
I would still leave "Once you activate your environment" to emphasize it is a required step.